### PR TITLE
Remove cyclic dependency and allow to pass Parallelism as an argument

### PR DESCRIPTION
**Pull request by r4rajat**
Signed-off-by: Rajat Gupta <rajat.gupta@veeam.com>

## Change Overview

- Remove cyclic dependency from package kopia/cmd
- Allow users to pass Parallelism in SnapshotCreateCommandArgs instead from Environmental Variables

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [x] :rainbow: Refactoring (no functional changes, no api changes)
- [x] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues 

- fixes #issue-number

## Test Plan

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

**Review by ankitjain235 — APPROVED** [review]

File: pkg/kopia/command/snapshot.go [hunk 1]
```diff
@@ -18,24 +18,25 @@ import (
...
 	"time"
 
-	"github.com/kanisterio/kanister/pkg/kopia"
 	"github.com/kanisterio/kanister/pkg/utils"
 )
 
 const (
-	requireLogLevelInfo = true
+	requireLogLevelInfo        = true
+	ManifestTypeSnapshotFilter = "type:snapshot"
```
Comment (PrasadG193) [inline pkg/kopia/command/snapshot.go:26]: No need to export the const if not being used outside this package.
```suggestion
	manifestTypeSnapshotFilter = "type:snapshot"
```

File: pkg/kopia/command/helpers.go [hunk 2]
```diff
@@ -15,26 +15,48 @@
...
+	// DefaultDataStoreGeneralContentCacheSizeMB is the default content cache size for general command workloads
+	DefaultDataStoreGeneralContentCacheSizeMB = 0
+	// DataStoreRestoreContentCacheSizeMBVarName is the name of the environment variable that controls
+	// kopia content cache size for restore workloads
+	DataStoreRestoreContentCacheSizeMBVarName = "DATA_STORE_RESTORE_CONTENT_CACHE_SIZE_MB"
+	// DefaultDataStoreRestoreMetadataCacheSizeMB is the default metadata cache size for restore workloads
+	DefaultDataStoreRestoreMetadataCacheSizeMB = 500
+	// DataStoreRestoreMetadataCacheSizeMBVarName is the name of the environment variable that controls
+	// kopia metadata cache size for restore workloads
+	DataStoreRestoreMetadataCacheSizeMBVarName = "DATA_STORE_RESTORE_METADATA_CACHE_SIZE_MB"
```
Comment (PrasadG193) [inline pkg/kopia/command/helpers.go:42]: Are these consts being used outside this package? If not, lets not export them.

Comment (r4rajat) [reply pkg/kopia/command/helpers.go:42]: Okay

File: pkg/kopia/command/snapshot.go (hunk 1 above)
Comment (r4rajat) [reply pkg/kopia/command/snapshot.go]: Done !

**Review by PrasadG193 — APPROVED** [review]

**Pull request merged**